examples/skywalking-tracing/verify.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+export NAME=skywalking
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+# Waiting SkyWalking server to be ready.
+sleep 60
+
+run_log "Test connection"
+responds_with \
+    "Hello from behind Envoy (service 1)!" \
+    http://localhost:8000/trace/1
+
+run_log "Test stats"
+responds_with \
+    "tracing.skywalking.segments_sent: 1" \
+    http://localhost:8001/stats
```

Comment:
this needs some more tests

firstly it needs to test that the ui `responds_with` or somesuch

that test alone will not pick up the problems im seeing locally you also need  another test to

http://localhost:8080/graphql

that `POST`s : 

```
{"query":"query queryOAPTimeInfo {\n    getTimeInfo {\n      timezone\n      currentTimestamp\n    }\n  }","variables":{}}
```

as the browser does when it fails